Added Nuclei

ehsandeep opened the pull request.

jamiebuilds commented on communities.json:
```diff
@@ -771,6 +771,13 @@
+    {
+      "logo": "nuclei-logo.png",
+      "title": "Nuclei",
+      "quote": "We connect with our community using discord server",
```
Just a small grammatical fix: 
```suggestion
      "quote": "We connect with our community using our Discord server",
```

jamiebuilds commented:
Turned on some features for you:

- [Vanity Invite URL](https://support.discord.com/hc/en-us/articles/115001542132-Server-Vanity-URLs)
- [Server Banner & Invite Splash](https://support.discord.com/hc/en-us/articles/360028716472-Server-Banner-Background-Invite-Splash-Image)

Do you want to update your invite code to something like `discord.gg/<yourprojectname>`?

ehsandeep commented:
@jamiebuilds thank you so much.

> Do you want to update your invite code to something like discord.gg/<yourprojectname>?

If something like this is possible `discord.gg/projectdiscovery`, or `discord.gg/projectdiscovery.io` Yes, please.

jamiebuilds commented:
I cannot update your vanity url for you, you'll need to do it in the app: https://support.discord.com/hc/en-us/articles/115001542132-Server-Vanity-URLs

ehsandeep commented:
Hi @jamiebuilds, 

Thank you, I've updated it from my side, and apologies for the double PR, I've created another PR here https://github.com/discord/discord-open-source/pull/330 as I've deleted my old branch, let me know if you want to close this PR.
